Magento_LayeredNavigation/templates/layer/view.phtml

```diff
@@ -47,21 +47,28 @@
                                 filter
                             "
                         >
-                            <div
+                            <button
+                                type="button"
                                 class="dropdown-list__label filter__label"
-                                aria-controls="checkbox"
+                                aria-controls="filter-<?= __($filter->getName()) ?>"
+                                aria-expanded="false"
                             >
-                                <?= $block->escapeHtml(__($filter->getName())) ?>
-                                <svg class="icon dropdown-list__icon">
+                                <h3 class="filter__label-title">
+                                    <?= $block->escapeHtml(__($filter->getName())) ?>
+                                </h3>
+                                <svg class="icon dropdown-list__icon" role="img">
```

Comment:
it is icon important for the user? maybe will be better to hide it from screen reader users with `role="presentation"` or `aria-hidden` ?

Reply:
ok, I changed from `role="img"` to `role="presentation"`

Reply:
👍